aikau/src/main/resources/alfresco/lists/views/layouts/Grid.js

```diff
@@ -189,6 +193,35 @@ define(["dojo/_base/declare",
          {
             focusedChild.blur();
          }
+         domClass.remove(focusedChild.domNode, "alfresco-lists-views-layouts-Grid--focused");
+      },
+
+      /**
+       * This function ensures that the widget requested to be focused has a focus function
+       * and if so calls the "focusChild" function provided by the _KeyNavContainer. Otherwise
+       * it manually takes case of setting the focus.
+       *
+       * @instance
+       * @param {object} widget The widget to focus
+       * @since 1.0.43
+       */
+      focusOnCell: function alfresco_lists_views_layouts_Grid__focusOnCell(widget) {
+         if (typeof widget.focus === "function")
+         {
+            this.focusChild(widget);
+         }
+         else
+         {
+            if(this.focusedChild && widget !== this.focusedChild){
+               this._onChildBlur(this.focusedChild);  // used to be used by _MenuBase
+            }
+            if (widget.domNode)
+            {
+               domAttr.set(widget.domNode, "tabIndex", this.tabIndex); // for IE focus outline to appear, must set tabIndex before focus
+               focusUtil.focus(widget.domNode);
+            }
+         }
+         domClass.add(widget.domNode, "alfresco-lists-views-layouts-Grid--focused");
```

Comment:
Are we really changing the state of the grid at this point? The state applies to the element that it follows. Would it be more appropriate to have `alfresco-lists-views-layouts-Grid__cell--focused`?